config/routes.rb

```diff
@@ -16,6 +16,7 @@
   post '/orders/:id', to: 'orders#create'
   post '/add_item', to: 'carts#add_item'
   post '/subtract_item', to: 'carts#subtract_item'
+  post '/remove_item', to: 'carts#remove_item'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.